integration-tests/build_run_cluster.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo $DRUID_INTEGRATION_TEST_OVERRIDE_CONFIG_PATH
+
+export DIR=$(cd $(dirname $0) && pwd)
+export HADOOP_DOCKER_DIR=$DIR/../examples/quickstart/tutorial/hadoop/docker
+export DOCKERDIR=$DIR/docker
+export SERVICE_SUPERVISORDS_DIR=$DOCKERDIR/service-supervisords
```

Comment:
Do we still need all of these variables?
I think SERVICE_SUPERVISORDS_DIR, SUPERVISORDIR, RESOURCEDIR can be reomved since the yml doesn't use them

Reply:
Probably we do not need all of them, need to check

Reply:
removed not need variables